### DISCUSSION TITLE
Auto change skins depending on which mode you play

### DISCUSSION THREAD

**Discussion by Shavixinio** [ppy/osu#29040] [Ideas] [answered]
Despite mainly sticking to one game mode, I find myself switching from game mode to game mode (for example from Taiko to Mania), and I'm bad at combining 2 skins that were made for specific game modes into one, so each time I switch a game mode, I have to switch skins, too, which is an annoying and time-consuming process in the long run.

An option to set specific skins for specific game modes would be really helpful, so I don't have to go to settings to each time change the skin.

**Comment by Joehuu** [comment, marked as answer]
This is a duplicate of https://github.com/ppy/osu/discussions/28695. Please upvote that instead.
